src/diagnostics/core_diagnostics.jl

```diff
@@ -733,3 +733,315 @@ add_diagnostic_variable!(
+    error_diagnostic_variable("dsevi", model)
+
+function compute_dsevi!(
+    out,
+    state,
+    cache,
+    time,
+    moisture_model::T,
+) where {T <: Union{EquilMoistModel, NonEquilMoistModel}}
+    thermo_params = CAP.thermodynamics_params(cache.params)
+    if isnothing(out)
+        out = zeros(axes(Fields.level(state.f, half)))
+
+        cp = CAP.cp_d(cache.params)
+        z = Fields.coordinate_field(state.c.ρ).z
+        g = CAP.grav(cache.params)
+        Operators.column_integral_definite!(
+            out,
+            LB.@lazy_broadcasted @. (
+                state.c.ρ * (
+                    cp *
+                    TD.air_temperature(thermo_params, cache.precomputed.ᶜts) +
+                    g * z
+                )
+            )
+        )
+        return out
+    else
+        cp = CAP.cp_d(cache.params)
+        z = Fields.coordinate_field(state.c.ρ).z
+        g = CAP.grav(cache.params)
+
+        Operators.column_integral_definite!(
+            out,
+            LB.@lazy_broadcasted @. (
+                state.c.ρ * (
+                    cp *
+                    TD.air_temperature(thermo_params, cache.precomputed.ᶜts) +
+                    g * z
+                )
+            )
+        )
+    end
+end
+########### Temporary fix ##############
+import ClimaCore: Operators, RecursiveApply, Fields
+function Operators.column_integral_definite!(
+    ∫field,
+    ᶜinput,
+    init = RecursiveApply.rzero(eltype(∫field)),
+)
+    ᶜinput_times_Δz = Base.Broadcast.broadcasted(
+        RecursiveApply.:⊠,
+        ᶜinput,
+        Fields.Δz_field(axes(ᶜinput)),
+    )
+    Operators.column_reduce!(RecursiveApply.:⊞, ∫field, ᶜinput_times_Δz; init)
+end
+########### Temporary fix #############
+
+add_diagnostic_variable!(
+    short_name = "dsevi",
+    long_name = "Dry Static Energy Vertical Integral",
+    standard_name = "dry_static_energy_vertical_integral",
+    units = "",
+    comments = """
+    """,
+    compute! = compute_dsevi!,
+)
+
+###
+# column integrated cloud fraction (1d)
+###
+compute_clvi!(out, state, cache, time) =
+    compute_clvi!(out, state, cache, time, cache.atmos.moisture_model)
+compute_clvi!(_, _, _, _, model::T) where {T} =
+    error_diagnostic_variable("clvi", model)
+
+function compute_clvi!(
+    out,
+    state,
+    cache,
+    time,
+    moisture_model::T,
+) where {T <: Union{EquilMoistModel, NonEquilMoistModel}}
+    thermo_params = CAP.thermodynamics_params(cache.params)
+    if isnothing(out)
+        out = zeros(axes(Fields.level(state.f, half)))
+        z =
+            Fields.coordinate_field(
+                cache.precomputed.cloud_diagnostics_tuple.cf,
+            ).z
+        Operators.column_integral_definite!(
+            out,
+            LB.@lazy_broadcasted @. (
+                ifelse.(
+                    cache.precomputed.cloud_diagnostics_tuple.cf .> 0.0f0,
+                    1.0f0,
+                    0.0f0,
+                ) * (z .< 4000)
```

Comment:
We should not make it integral below 4km for the general purpose. If you do want it, you can use a more specific name, e.g `clvi4000`. But I don't think the vertical integral of cloud fraction is a very useful diagnostics - What do you use it for?

Reply:
Ok - I will change it to the whole profile. This was one of the metrics that tapio suggested I try for calibration